import-roles/import-roles.go

```diff
@@ -9,10 +9,12 @@ import (
 
 	"github.com/ONSdigital/log.go/log"
 	"github.com/globalsign/mgo"
+	uuid "github.com/satori/go.uuid"
 )
 
 //Role represents a role that will be stored in mongo
 type Role struct {
+	ID          string   `bson:"_id" json:"id"`
```

Comment:
Can we use the role type defined in `models/role.go` rather than having it declared here?